BTCPayServer/Controllers/UIWalletsController.cs

```diff
@@ -259,20 +259,31 @@ public async Task<IActionResult> ListWallets()
             WalletId walletId,
             string? labelFilter = null,
             int skip = 0,
-            int count = 50
+            int count = 50,
+            TimeSpan? interval = null
```

Comment:
We can simplify it quite a bit in the other parts also when we use an `int` here and also rename it, to reveal the intention better: 
  
```suggestion
            int days = 30
```

Reply:
Changed it to days. LOT CLEANER!! Thank you for the suggestion!